ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -55,3 +97,40 @@ def main():
         procs.append(proc)
     for p in procs:
         p.join()
+
+
+def aws_cleanup():
+    parser = argparse.ArgumentParser(description='Cleanup AWS Resource')
+    parser.add_argument(
+        '--hours',
+        type=int,
+        nargs=1,
+        action='append',
+        required=True,
+        help="Cluster running time in hours"
```

Comment:
This is sort of vague, and doesn't mention that it is the maximum age of the cluster you are allowing. Maybe we can be more specific here:

"Maximum running time of the cluster (in hours). Clusters older than this will be deleted"

WDYT?

Reply:
thanks! changed